commit 09c043e560b69c5ba803f0b35eea75fb8f1ee85e
Author: Zoltan Varga <vargaz@gmail.com>
Date:   Mon Sep 24 21:54:47 2018 +0200

    Add '___cxa_is_pointer_type', '___cxa_can_catch' to the list of exported functions when exception catching is enabled.

diff --git a/emcc.py b/emcc.py
index bb2889fdb..65c71a174 100755
--- a/emcc.py
+++ b/emcc.py
@@ -1134,6 +1134,8 @@ There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR P
       if shared.Settings.DEMANGLE_SUPPORT:
         shared.Settings.EXPORTED_FUNCTIONS += ['___cxa_demangle']
         forced_stdlibs += ['libcxxabi']
+      if shared.Settings.DISABLE_EXCEPTION_CATCHING != 1:
+        shared.Settings.EXPORTED_FUNCTIONS += ['___cxa_is_pointer_type', '___cxa_can_catch']
 
       if not shared.Settings.ONLY_MY_CODE:
         if type(shared.Settings.EXPORTED_FUNCTIONS) in (list, tuple):
